### DISCUSSION TITLE
Really strange network connection issue

### DISCUSSION THREAD

**Discussion by ceejayemm** [rern/rAudio#1777] [Bug Reports]
RPI 3B+, rAudio 202409141 updated to UPDATE branch (15 Sept 2024 for resolution to #1762)

I am not sure when this started or if in fact it is related to applying the Update branch at all.

I have been using the Android app to connect to rAudio for a long time with no issues, until today.  This morning the Android app refused to connect to my rAudio system via its IP number.  The IP number displayed in the app was correct but no matter what I did it would not connect.  This afternoon with some time to look at this I have found that IP access from the Android app was still broken. From my laptop I pinged the rAudio IP number which came back as 'Destination Host Unreachable'.  Additionally, I also could not connect from a laptop based browser via IP Number.  I pinged its network name and this came back with a successful connection and I could connect via the web browser via the system's network name.  Having made the named connection via my laptop I found I could then connect via the IP number on both my laptop and the Android App.Once the connection was made, either from web browser or app, rAudio seems to work as I would expect.

I looked at the network connections on the RPi and found:

```
RAConservatory:~  #  ip a
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default qlen 1000
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
    inet 127.0.0.1/8 scope host lo
       valid_lft forever preferred_lft forever
2: enu1u1u1: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc fq_codel state DOWN group default qlen 1000
    link/ether b8:27:eb:24:d8:7d brd ff:ff:ff:ff:ff:ff
3: wlan0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc fq_codel state UP group default qlen 1000
    link/ether b8:27:eb:71:8d:28 brd ff:ff:ff:ff:ff:ff
    inet 192.168.86.33/24 brd 192.168.86.255 scope global dynamic noprefixroute wlan0
       valid_lft 75840sec preferred_lft 61548sec
RAConservatory:~  #  
```

I normally connect to this system via WiFi (wlan0 - 192.168.86.33) and I have no idea where the 'enu1u1u1' connection has appeared from.  I do not have a wired connection to the system. I have not made any changes to my network for quite some time and am at a loss as to why this problem has occurred now.

Any explanations anybody ?

Many thanks

Chris

**Comment by kan-ibal** [comment]
The 'enu1u1u1' is not a connection but an ethernet interface only that is ready to connect to it. You can see NO-CARRIER hence no inet is configured. It is not a cause of your issues. It looks like a router hiccups for me. I would restart a wifi router.
